src/mina-signer/tests/rosetta.test.ts

```diff
@@ -1,11 +1,13 @@
 import Client from '../dist/node/mina-signer/MinaSigner.js';
+import { signatureFromHex, signatureToHex } from '../dist/node/mina-signer/src/rosetta.js';
 
 describe('Rosetta', () => {
   let client: Client;
 
+  const rosettaTnxMockSignature = '389ac7d4077f3d485c1494782870979faa222cd906b25b2687333a92f41e40b925adb08705eddf2a7098e5ac9938498e8a0ce7c70b25ea392f4846b854086d43';
   const signedRosettaTnxMock = `
   {
-    "signature": "389ac7d4077f3d485c1494782870979faa222cd906b25b2687333a92f41e40b925adb08705eddf2a7098e5ac9938498e8a0ce7c70b25ea392f4846b854086d43",
+    "signature": "${rosettaTnxMockSignature}",
```

Comment:
bug
```suggestion
    "signature": `${rosettaTnxMockSignature}`,
```

Reply:
how so? the entire blob is in backticks. here is the output of `console.log(rosettaTnxMockSignature)` left as is:

```
      {
        "signature": "389ac7d4077f3d485c1494782870979faa222cd906b25b2687333a92f41e40b925adb08705eddf2a7098e5ac9938498e8a0ce7c70b25ea392f4846b854086d43",
        "payment": {
          "to": "B62qnzbXmRNo9q32n4SNu2mpB8e7FYYLH8NmaX6oFCBYjjQ8SbD7uzV",
          "from": "B62qnzbXmRNo9q32n4SNu2mpB8e7FYYLH8NmaX6oFCBYjjQ8SbD7uzV",
          "fee": "10000000",
          "token": "1",
          "nonce": "0",
          "memo": null,
          "amount": "1000000000",
          "valid_until": "4294967295"
        },
        "stake_delegation": null
      }
```

Reply:
Right you don't need the backticks. But read your code again - it has "${blub}" which doesn't work

Reply:
you need the backticks for interpolation.

```
// correct this alone will not work
"${blub}"

// however inside the backticks it does
`"${blub}"`

// the double quotes are needed for that to be represented as a json string otherwise it'll appear to be an int - removing them would cause the later calls in the test to JSON.parse to fail

Reply:
my bad - I didn't see the entire blob is in backticks 😅